modules/nf-core/ensemblvep/download/main.nf

```diff
@@ -0,0 +1,43 @@
+process ENSEMBLVEP_DOWNLOAD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::ensembl-vep=108.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ensembl-vep:108.2--pl5321h4a94de4_0' :
+        'quay.io/biocontainers/ensembl-vep:108.2--pl5321h4a94de4_0' }"
+
+    input:
+    tuple val(meta), val(assembly), val(species), val(cache_version)
+
+    output:
+    tuple val(meta), path("vep_cache"), emit: cache
+    path "versions.yml"               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    vep_install \\
+        --CACHEDIR vep_cache \\
+        --SPECIES $species \\
+        --ASSEMBLY $assembly \\
+        --CACHE_VERSION $cache_version \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        ensemblvep: \$( echo \$(vep --help 2>&1) | sed 's/^.*Versions:.*ensembl-vep : //;s/ .*\$//')
+    END_VERSIONS
+    """
+
+    stub:
+    """
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        ensemblvep: \$( echo \$(vep --help 2>&1) | sed 's/^.*Versions:.*ensembl-vep : //;s/ .*\$//')
+    END_VERSIONS
+    """
```

Comment:
Maybe make an empty directory called `vep_cache` in the stub (I think the stub run will fail otherwise)

Reply:
I knew I forgot something